linode/manager
# feat: [M3-6945] – Support VPC and Subnet events

## Description 📝
Add the following VPC and Subnet events to `eventMessageGenerator.ts`:
- `vpc_create`
- `vpc_update`
- `vpc_delete`
- `subnet_create`
- `subnet_update`
- `subnet_delete`

~~WIP until API clarifies whether the event objects will be fleshed out for these events.~~

Additionally, I updated the routing from `/vpc` to `/vpcs` to be consistent with other entities like `/linodes`, `/databases`, etc.

## Preview 📷
Screenshots attached to the ticket

## How to test 🧪
- Confirm that VPC and subnet events appear as expected with proper links in Cloud
- Confirm all VPC routing functions as it should
